cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -153,6 +153,12 @@ if (ORC_PACKAGE_KIND STREQUAL "conan")
   add_resolved_library (orc_snappy ${Snappy_LIBRARIES} ${Snappy_INCLUDE_DIR})
   list (APPEND ORC_SYSTEM_DEPENDENCIES Snappy)
   list (APPEND ORC_INSTALL_INTERFACE_TARGETS "$<INSTALL_INTERFACE:Snappy::snappy>")
+elseif (ORC_PACKAGE_KIND STREQUAL "vcpkg")
+  find_package(Snappy CONFIG REQUIRED)
+  add_library (orc_snappy INTERFACE IMPORTED)
+  target_link_libraries(orc_snappy INTERFACE Snappy::snappy)
```

Comment:
It doesn't need to call `target_include_directories` because `Snappy::snappy` already exported the headers?

Reply:
@wgtmac  Actually, we don't call target_include_directories here, right?